src/mmpars.c

```diff
@@ -3898,140 +3900,140 @@ vstring outputStatement(long stmt, flag reformatFlag) {
+    // g_Statement[stmt].scope is at start of stmt not end; adjust $}
     if (g_Statement[stmt].type == rb_) indent = indent - INDENT_INCR;
 
-    /* Untab the label section */
-    if (strchr(labelSection, '\t') != NULL) { /* Only if has tab (speedup) */
-      let(&labelSection, edit(labelSection, 2048/*untab*/));
+    // Untab the label section
+    if (strchr(labelSection, '\t') != NULL) { // Only if has tab (speedup)
+      let(&labelSection, edit(labelSection, 2048)); // untab
     }
-    /* Untab the math section */
-    if (strchr(mathSection, '\t') != NULL) { /* Only if has tab (speedup) */
-      let(&mathSection, edit(mathSection, 2048/*untab*/));
+    // Untab the math section
+    if (strchr(mathSection, '\t') != NULL) { // Only if has tab (speedup)
+      let(&mathSection, edit(mathSection, 2048)); // untab
     }
 
-    /* Reformat the label section */
+    // Reformat the label section
 
-    /* Remove spaces a end of line */
-    /* This is a pretty inefficient loop, but hopefully lots of spaces
-       at end of lines is a rare occurrence */
+    // Remove spaces a end of line.
+    // This is a pretty inefficient loop, but hopefully lots of spaces
+    // at end of lines is a rare occurrence.
     while (1) {
       pos = instr(1, labelSection, " \n");
       if (pos == 0) break;
       let(&labelSection, cat(left(labelSection, pos - 1),
           right(labelSection, pos + 1), NULL));
     }
 
-    /* Don't allow more than 2 consecutive blank lines */
+    // Don't allow more than 2 consecutive blank lines
     while (1) {
-      /* Match to 3 or more blank lines */
+      // Match to 3 or more blank lines
       pos = instr(1, labelSection, "\n\n\n\n");
-      /* If it matches, remove one of the \n and restart loop */
+      // If it matches, remove one of the \n and restart loop
       if (pos == 0) break;
       let(&labelSection, cat(left(labelSection, pos - 1),
           right(labelSection, pos + 1), NULL));
     }
 
     switch (g_Statement[stmt].type) {
-      case lb_: /* ${ */
-      case rb_: /* $} */
-      case v_: /* $v */
-      case c_: /* $c */
-      case d_: /* $d */
-        /* These 5 cases are for keywords that don't have labels, so that
-           labelSection is simply the text before the keyword. */
-
-        /* Strip any trailing spaces */
-        let(&labelSection, edit(labelSection, 128 /*trailing spaces*/));
-        slen = (long)strlen(labelSection); /* Save to avoid recomputing */
-        /* See if last character is \n; if not, add it */
-        /* (If there's no text - just spaces - they've been stripped, and
-            leave labelSection as an empty string.) */
-        /* We use slen - 1 because native C strings start at index 0. */
+      case lb_: // ${
+      case rb_: // $}
+      case v_: // $v
+      case c_: // $c
+      case d_: // $d
+      // These 5 cases are for keywords that don't have labels, so that
+      // labelSection is simply the text before the keyword.
```

Comment:
```suggestion
        // These 5 cases are for keywords that don't have labels, so that
        // labelSection is simply the text before the keyword.
```